reacthookspring/webclient/app/src/app/commons/form/EntityDetail.jsx

```diff
@@ -94,10 +94,34 @@ function EntityDetail(props) {
                                     </FormControl>
                                 </div>
                             );
+                        } else if (isEnum(field.type)) {
```

Comment:
Die einzelnen return values müssen noch in seperate Funktionen ausgelagert werden (für das nächste refactoring)